### DISCUSSION TITLE
Installed renewable capacity in Italy.

### DISCUSSION THREAD

**Discussion by Raffaelemzz** [electricitymaps/electricitymaps-contrib#5363] [General]
The data visible on Electricitymap are obsolete.
Here every month Terna publishes an update on the italian photovoltaic, wind, hydro, and geothermal/bioenergy generation capacity:
https://www.terna.it/it/sistema-elettrico/dispacciamento/fonti-rinnovabili
It is possible to see data for each individual Italian region.
As of March 31, 2023, Italy has:
**26.11 GW** Photovoltaic
11.99 GW wind power
21.71 GW hydroelectric
4.91   GW geothermal

**Comment by VIKTORVAV99** [comment]
Hi @Raffaelemzz!

It looks like this source only have capacities for Italy as a whole and administrative areas, not the electricity bidding areas. This means we can't really use this particular source for our Italian capacities but I'll see if I can find some updated ones elsewhere.

**Reply by Raffaelemzz** [reply]
First of all, I would like to mention that Calabria and South Italy are two separate bidding areas.

Apparently bidding areas correspond to a grouping of administrative regions.

https://www.mercatoelettrico.org/en/mercati/mercatoelettrico/zone.aspx

• North Italy: Val D’Aosta, Piemonte, Liguria, Lombardia, Trentino, Veneto, Friuli Venezia Giulia, Emilia Romagna
• Central North Italy: Toscana, Marche
• Central South Italy: Lazio, Abruzzo, Campania, Umbria
• South Italy: Molise, Puglia, Basilicata
• Calabria: Calabria
• Sicily: Sicilia
• Sardinia: Sardegna

So I really believe that the data on Terna's website are safely usable to update Electricity Maps

**Reply by Raffaelemzz** [reply]
I think there is a problem.
![IMG_8090](https://github.com/electricitymaps/electricitymaps-contrib/assets/122230113/842fc268-8ee7-4045-8259-10211ee77f78)

**Reply by VIKTORVAV99** [reply]
This is a issue with the source data (the data that is reported to ENTSO-E) but I'll make a note of it and add some basic validation to the parser to discard these datapoints. This should correct itself when it's fixed in ENTSO-E but I'll keep an eye out if we need to re-fetch the data.

**Reply by VIKTORVAV99** [reply]
> First of all, I would like to mention that Calabria and South Italy are two separate bidding areas.
> 
> Apparently bidding areas correspond to a grouping of administrative regions.
> 
> https://www.mercatoelettrico.org/en/mercati/mercatoelettrico/zone.aspx
> 
> • North Italy: Val D’Aosta, Piemonte, Liguria, Lombardia, Trentino, Veneto, Friuli Venezia Giulia, Emilia Romagna • Central North Italy: Toscana, Marche • Central South Italy: Lazio, Abruzzo, Campania, Umbria • South Italy: Molise, Puglia, Basilicata • Calabria: Calabria • Sicily: Sicilia • Sardinia: Sardegna
> 
> So I really believe that the data on Terna's website are safely usable to update Electricity Maps

Ah that's good to know, then we can defiantly update the capacities. And we are aware Calabria and South Italy are separate bidding areas and we purposefully merged them back in the day to keep our historical data intact. I'll check in with the team if we can split them up soon.

**Reply by Raffaelemzz** [reply]
Almost a year later, the data on installed renewable capacity has still not been updated.
Meanwhile, the Italian bidding areas have changed. Umbria is now part of the 'centre south'.
![image](https://github.com/electricitymaps/electricitymaps-contrib/assets/122230113/e0b7994a-481e-4454-9248-54fa68fe9c6f)
As of November 30, 2023, Italy has:
**29.56 GW** Photovoltaic
12.32 GW wind power
21.73 GW hydroelectric
4.95 GW geothermal

As mentioned before, on Terna's website there is an excel with the capacity of each Italian region.
https://www.terna.it/it/sistema-elettrico/dispacciamento/fonti-rinnovabili

**Reply by Raffaelemzz** [reply]
The delay in updating this data and the correction on Italian areas is concerning.
Italy now has over 31.5 GW from photovoltaics (+26%).
**Umbria** is part of the **Central South Italy**, **not**  **Central North Italy** as wrongly shown on your site.
Calabria is a billing area of its own.

Considering also the overestimated emissions for unknown sources, the accuracy of ElectricityMaps for Italy is debatable.

**Reply by Raffaelemzz** [reply]
@VIKTORVAV99 @ Why has the data not been updated in almost 2 years?
Italy has now reached **36.4 GW** of photovoltaics (while according to ElectricityMaps they are still only **22.6 GW**).
You can find the most up-to-date data by zone here:
https://dati.terna.it/en/generation#installed-renewables
![image](https://github.com/user-attachments/assets/96228efb-3d53-4230-bf33-450d6a54077c)

Umbria is part of the Central South Italy, not Central North Italy as wrongly shown on your site.
![image](https://github.com/user-attachments/assets/d0cf0443-1cf5-49b4-a355-1dce09f5c9d8)
